openmetadata-service/src/main/java/org/openmetadata/service/security/DefaultAuthorizer.java

```diff
@@ -334,6 +328,14 @@ public static User addOrUpdateBotUser(User user, OpenMetadataApplicationConfig o
     return addOrUpdateUser(user);
   }
 
+  private static JWTAuthMechanism buildJWTAuthMechanism(OpenMetadataJWTClientConfig jwtClientConfig, User user) {
+    return jwtClientConfig == null || jwtClientConfig.getJwtToken() == null
```

Comment:
we should add an "" check on the JWT token as well, by default we get airflowConfig as not null with empty configurations so it will initialise it with an empty token I think